### ISSUE TITLE
ThumbnailProcessor mit Webp-Unterstützung with cloudimg

### ISSUE THREAD

**Issue by Oliverhorn** [FriendsOfShopware/FroshPlatformThumbnailProcessor#95]
### Question

Hi,
i'm installed that plugin for shopware 6 and already had cloudimg configured. I'm confused about the correct way to use it.
So i have some questions about it. I'm running a dev environment for shopware6, no production env.

1. After installing and activating the app, the pictures are correctly linked to cloudimg.io with my token. In the media folder in shopware, i can click on the link for the picture and the picture is shown loaded from cloudimg.

2. Sometimes, the pictures aren't shown from cloudimg. Only the placeholder from cloudimg. Why?
3. After switching of the app, the links for the media aren't updated to the former link from shopware without the cloudimg token. why?
4. How can i easily switch between using cloudimg and using not cloudimg? Is there any way with your app? Do i have to create or update after switching off your app?
5. When i add new pictures for a new product, the thumbnails wont be created new and the link with the cloundimg token shows only the placeholder picture.

So, basically i do not know, how to use your app correctly. Hopefully, you can help me and give me a tutorial to use it in the correct way.

Thanks a lot!
Oliver

**Comment by tinect** [comment]
Hey,

Thank you for contacting us.

1. what is the question?
2. check the link, seems clouding-related.
3. You have to clear the cache and generate thumbnails by shopware cli: media:thumbnails-generate 
4. See 3.
5. Could you check the image is accessible by cloudimg? Regarding point 2 is sounds like there is a problem from accessing the image. (DevTools f.e.)

Additionally, you mean cloudimage?

**Comment by Oliverhorn** [comment]
Hi,
yes, i mean cloudimage.
The question is, why are the pictures not shown? Only the placeholder picture from cloudimage?
The pattern is correctly set, with my token and my url. Like that: https://TOKEN.cloudimg.io/www.myshop.com/{mediaPath}?w={width}&func=bound&org_if_sml=1

Is there's something wrong with cloudimage? Are there adjustments with cloudimage i have to do?

**Comment by tinect** [comment]
No, there is nothing special to set up. Just exchange Token and the url.

could you please contact me by PM in shopware slack? So you can send me your shops links and I can check there for the url.

**Comment by tinect** [comment]
Solution: use the correct domain where your images are accessible.
